stories/cli/try.fmf

```diff
@@ -145,7 +145,7 @@ link:
 
         link:
           - implemented-by: /tmt/trying.py
-          - implemented-by: /tmt/steps/prepare/feature.py
+          - implemented-by: /tmt/steps/prepare/feature/__init__.py
```

Comment:
`feature/epel.py` would be better than `feature/__init__.py`, now when we have actual plugins for features.

Reply:
Hm, yes it is, I will change it